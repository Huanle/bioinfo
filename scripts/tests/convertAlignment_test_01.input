3 110
seq7 ATGTTATCAGCTCTAGTCGACAAGTATCTCAACTACCAGGAACCTCCGAATCGACAACTT
seq8 ATGTTATCGGCTCTTGTAGAGAGACATCTCAGCTTCATAGAGTCTTCG------CAAC--
seq9 ATGTTATCAGCTCTAGTCGACAAATATCTGAACTACCAGGAACCTCCTGATCGACAACAC

CAACTTTCGGGTGAAGACTCCAGAATCGCCTACGACAACGGGTCGGACAG
------AGGGGTGAAGACTCGAGAATCGCTTACGACAACGGGTCGGACAG
CAGCTTCAAGGTGAGGCCACCAGAATCGCCTACGACAACGGCTCAGATAG
