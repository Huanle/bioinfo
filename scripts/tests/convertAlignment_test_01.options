--input convertAlignment_test_01.input --if phylip --of fasta
