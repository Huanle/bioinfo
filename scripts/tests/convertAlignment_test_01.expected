>seq7
ATGTTATCAGCTCTAGTCGACAAGTATCTCAACTACCAGGAACCTCCGAATCGACAACTT
CAACTTTCGGGTGAAGACTCCAGAATCGCCTACGACAACGGGTCGGACAG
>seq8
ATGTTATCGGCTCTTGTAGAGAGACATCTCAGCTTCATAGAGTCTTCG------CAAC--
------AGGGGTGAAGACTCGAGAATCGCTTACGACAACGGGTCGGACAG
>seq9
ATGTTATCAGCTCTAGTCGACAAATATCTGAACTACCAGGAACCTCCTGATCGACAACAC
CAGCTTCAAGGTGAGGCCACCAGAATCGCCTACGACAACGGCTCAGATAG
